.github/workflows/semantic-pull-request.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Lint PR'
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v4
```

Comment:
This will need security review

Reply:
This is settings about how we restricted actions on Github. Changeset has been signed off by AppSec <- https://github.com/aws-amplify/amplify-ui/settings/actions